lib/gooddata/models/project.rb

```diff
@@ -862,7 +862,7 @@ def data_permissions(id = :all)
     # Deletes project
     def delete
       fail "Project '#{title}' with id #{uri} is already deleted" if deleted?
-      client.delete(uri)
+      client.delete(uri, tries: 2)
```

Comment:
why change code? we should change test code only

Reply:
yes I will update test code only